community.md

```diff
@@ -1,9 +1,20 @@
 # Community
 
+## Slack
+
 The OpenFisca community gathers around a [Slack space](https://openfisca.slack.com), which you can ask to join by sending a mail to [contact@openfisca.org](mailto:contact@openfisca.org?subject=Slack).
 
 This space provides both community and official support, and centralises all countries’ channels.
 
+### Channels and naming conventions
+
+In order to increase discoverability of channels and ease navigation, the following prefixes are used:
+
+- `of-` channels are about quick discussions and requests for help on a technical module. They don’t replace GitHub issues and Pull requests as the mean to reach consensus.
```

Comment:
I find this channel description a bit too abrupt : We are tryung to foster the exchanges between contributive teams and I feel the last phrase may inhibit discussions.
How about : 

- `of-` channels allow quick discussions and asks for help on shared technical modules (Core, Legislation Explorer ....). To reach concensus or make decisions on those modules, we use  GitHub issues and Pull Requests on dedicated repos.